### DISCUSSION TITLE
Mercury theme – Front page -> Filter by: Maintag -> Style of the label text is broken in a way

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1448] [Bug Report] [answered]
### Operating system

macOS Sonoma

### Publii version

45.2

### Issue type

Free theme

### Bug description

Assuming, that it is a bug and not a feature: In Mercury 2.7.0.0 one can add a label for the MainTag filter. This label text is not in line with the rest of the Filter:
![Bildschirmfoto 2024-03-22 um 09 18 21](https://github.com/GetPublii/Publii/assets/17292904/93c03a01-f1ed-43bb-a8d7-cb8dcde33040)

**Comment by bobmitro** [comment, marked as answer]
You can fix it via CSS.

**Reply by 3muchfun** [reply]
yes, sure, but I thought you maybe have intention to fix it our your side for all users. But, on the other hand, if it is designed as a feature, I will repair it on my own.
